frontend/public/components/error.tsx

```diff
@@ -84,25 +84,27 @@ export const ErrorBoundaryFallback: React.SFC<ErrorBoundaryFallbackProps> = (pro
   <div className="co-m-pane__body">
     <h1 className="co-m-pane__heading co-m-pane__heading--center">Oh no! Something went wrong.</h1>
     <div className="row">
-      <ExpandCollapse textCollapsed="Show Details" textExpanded="Hide Details">
-        <h3 className="co-section-heading-tertiary">{props.title}</h3>
-        <div className="form-group">
-          <label htmlFor="description">Description: </label>
-          <p>{props.errorMessage}</p>
-        </div>
-        <div className="form-group">
-          <label htmlFor="componentTrace">Component Trace: </label>
-          <div className="co-copy-to-clipboard__stacktrace-width-height">
-            <CopyToClipboard value={props.componentStack.trim()} />
+      <div className="col-xs-12">
```

Comment:
Do we need Bootstrap grid at all here? If we simply removed `<div className="row">`, is the bug fixed?

Reply:
That also works. Updated pr